cmd/ddev/cmd/get.go

```diff
@@ -89,6 +88,7 @@ ddev get --remove ddev-someaddonname
 func init() {
 	Get.Flags().Bool("list", true, `List available add-ons for 'ddev get'`)
 	Get.Flags().Bool("all", true, `List unofficial add-ons for 'ddev get' in addition to the official ones`)
+	_ = Get.Flags().MarkDeprecated("all", "this flag no longer has any effect. All add-ons are shown by default.")
```

Comment:
I recommend either _not_ deprecating here (this whole command is deprecated anyway), or else removing likes 38-40 (i.e. don't pass `--all` through since it doesn't do anything anymore).